easybuild/easyconfigs/p/PyQt/PyQt-x11-gpl-4.11.3-goolf-1.5.14.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMakePythonPackage'
+name = 'PyQt'
+version = 'x11-gpl-4.11.3'
```

Comment:
I'd prefer using `4.11.3` as a version, and include `x11-gpl` in `sources` and `source_urls` instead?